packages/mobile/src/send/saga.ts

```diff
@@ -178,6 +178,7 @@ function* sendPayment(
       amount: amount.toString(),
       currency,
     })
+    yield call(giveProfileAccess, [recipientAddress])
```

Comment:
Hm what do you mean by withdrawing CELO? From the CICO providers?

Reply:
No, I mean from the CELO tab, there's a withdraw feature where you can withdraw CELO to other addresses. Then, on the feed on the CELO tab you see `Received from ${address}` is we have no info from that address (there's no picture on the CELO feed). If we gave profile access when withdrawing, we'd be able to see the name of the sender instead of the address.

CELO withdrawals is not really targeted for p2p usage though (hence the confusing 'withdraw' verb used) so I'd understand not doing this, although I think it would be best to do it.

Reply:
@nityas @aleksey-nov what do you guys think?